iatidataquality/templates/surveys/survey.html

```diff
@@ -48,7 +48,7 @@
     </td>
     <td>{{workflow.Workflow.title}}</td>
     <td>
-		{% if workflow.Workflow.id <= survey.Workflow.id %}
```

Comment:
This is a bug, and was the motivation for this PR. Comparing IDs like this is not correct, since the value of workflow IDs is not an indication of ordering. It’s not really easy to do the right thing here by referring to `leadsto` either, though.